ui_framework/components/global_styles/mixins/_typography.scss

```diff
@@ -0,0 +1,17 @@
+// Some mixins that help us deal with browser scaling of text more consistantly.
+// Essentially, fonts across kui should scale agains the root html element, not
+// against parent inheritance.
+
+
+// Typography mixins
+
+@function convertToRem($size) {
+  @return #{$size / $kuiFontSize}rem;
+}
+
+// Spit out rem and px
+
+@mixin fontSize($size: $kuiFontSize) {
+  font-size: $size;
+  font-size: convert-to-rem($size);
```

Comment:
And why are we calling `convert-to-rem` when the name of the function is `convertToRem`?